packages/core/src/errorCollection.ts

```diff
@@ -13,12 +13,14 @@ export interface ErrorMessage {
     error: ErrorContext
     http?: HttpContext
   }
+  customerContext?: Context
+  savedGlobalContext?: Context
```

Comment:
I didn't want to modify the `ErrorMessage` too much, but if we follow the same logic used in `addUserAction`, we should [separate the `savedGlobalContext` from other props](https://github.com/DataDog/browser-sdk/blob/master/packages/rum/src/domain/lifeCycle.ts#L42-L45) like:

```ts
interface ErrorMessageWithGlobalContext {
  errorMessage: ErrorMessage,
  context?: Context
}
```

IMO, storing the global context in the error message is fine, and multiplying structure definitions makes it harder to understand.